kendo-ui/kendo-ui.d.ts

```diff
@@ -18290,6 +18290,7 @@ declare namespace kendo.ooxml {
         cells?: WorkbookSheetRowCell[];
         index?: number;
         height?: number;
+        type?: string;
```

Comment:
Should this type be `type?: "header" | "footer" | "groupHeader" | "groupFooter" | "data"  As indicated [here](http://docs.telerik.com/kendo-ui/controls/data-management/grid/excel-export#row-type)